docs/workloads/azure/backend/java/testing_java.md

```diff
@@ -111,20 +111,20 @@ To generate the Serenity report:
   <TabItem value="unix">
 
   ```bash
-  ./mvnw clean verify"
+  ./mvnw clean verify
   ```
 
   </TabItem>
   <TabItem value="windows">
 
   ```bash
-  mvnw.cmd clean verify"
+  mvnw.cmd clean verify
   ```
 
   </TabItem>
  </Tabs>
 
-The report can be viewed under `<PROJECT-NAME>/api-tests/target/site/serenity/index.html`.
+The report can be viewed under `./api-tests/target/site/serenity/index.html`.
 
 #### Code coverage
 
```

Comment:
Little passage here about what tool is being used for code coverage and how it works